### PR TITLE
Lms 2307 notifications will not send to email addresses

### PR DESCRIPTION
**Pull request by stephen-oh**
**Description**
- Notifications were not sending to email addresses in plugin config

**Related Issues**
- N/A

**PR Type**
- Fix

### CODE REVIEW

File: classes/task/sendfailednotifications_task.php [hunk 1]
```diff
@@ -183,32 +184,32 @@ public function execute() {
         $manager = get_log_manager();
         $store = new store($manager);
 
-        echo "In send failed notifications task execute".PHP_EOL;
+        echo "In send failed notifications task execute" . PHP_EOL;
```
Comment (lzabo) [inline classes/task/sendfailednotifications_task.php]: Could you add this into the string file?

File: lib.php [hunk 2]
```diff
@@ -75,15 +75,11 @@ function logstore_xapi_get_cohort_members($cohortids) {
         // Validate params.
         $cohort = $DB->get_record('cohort', array('id' => $cohortid), '*', MUST_EXIST);
         if (!empty($cohort)) {
-            $cohortmembers = $DB->get_records_sql("SELECT u.id, u.email
+            $cohortmembers = $DB->get_records_sql("SELECT u.*
                 FROM {user} u, {cohort_members} cm
                 WHERE u.id = cm.userid AND cm.cohortid = ?
                 ORDER BY lastname ASC, firstname ASC", array($cohort->id));
```
Comment (lzabo) [inline lib.php]: Could you reorganise this sql's coding style to follow Moodle docs?
https://docs.moodle.org/dev/SQL_coding_style

File: classes/task/sendfailednotifications_task.php (hunk 1 above)
Comment (gordonmacqueen-lp) [reply classes/task/sendfailednotifications_task.php]: Do we even need this line @mlynn-lp?

Comment (mlynn-lp) [reply classes/task/sendfailednotifications_task.php]: No it's just to show some output when running on the cli.

**Review by gordonmacqueen-lp — APPROVED** [review]

**Review by lzabo — APPROVED** [review]

**Pull request merged**